service/history/workflow/task_generator.go

```diff
@@ -193,7 +190,7 @@ func (r *TaskGeneratorImpl) GenerateWorkflowCloseTasks(
 				Version:     currentVersion,
 			},
 		)
-		if r.archivalQueueEnabled() {
+		if r.archivalEnabled() {
```

Comment:
so if we are deleting an open workflow, that workflow won't be archived right? just want to confirm my understanding.

Reply:
Yeah the manual delete skips archival.